Transcriptor.lua

```diff
@@ -1246,6 +1247,7 @@ local bwEvents = {
 }
 local dbmEvents = {
 	"DBM_Announce",
+	"DBM_Debug"
```

Comment:
Missing comma

Reply:
Dammit, so many typos in one simple addition. Sorry